628.max-product-of-three-numbers.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * https://leetcode.com/problems/maximum-product-of-three-numbers/description/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+
+var maximumProduct = function (nums) {
+  nums.sort((a, b) => b - a);
+  const positiveSum = nums[0] * nums[1] * nums[2];
+  const negativeTwo = nums[0] * nums[nums.length - 1] * nums[nums.length - 2];
+
+  if (positiveSum > negativeTwo) {
+    return positiveSum;
+  } else {
+    return negativeTwo;
```

Comment:
```suggestion
  positiveSum > negativeTwo ? return positiveSum : return negativeTwo;
```
You can also write like this.